cmd/bclient/bserver/bserver.go

```diff
@@ -9,14 +9,10 @@ import (
 )
 
 var (
-	ItemFetchStatus error
-
-	fileIDMutex sync.Mutex
-
-	fileIDList []fileIDStruct
-
-	bendoServer *string
-	RemoteJason *jason.Object
+	itemFetchStatus error = nil
+	fileIDMutex     sync.Mutex
+	fileIDList      []fileIDStruct
+	RemoteJason     *jason.Object
```

Comment:
These globals are horrible and I still hate them. FYI. but I'm not requiring them to be fixed now.